fenix/app/src/main/java/org/mozilla/fenix/settings/search/SearchEngineFragment.kt

```diff
@@ -32,6 +32,12 @@ class SearchEngineFragment : PreferenceFragmentCompat() {
             if (unifiedSearchUI) R.xml.search_settings_preferences else R.xml.search_preferences,
             rootKey,
         )
+
+        // Visibility should be set before the view has been created, to avoid visual glitches.
```

Comment:
```suggestion
        // Visibility should be set before the view has been created to avoid visual glitches.
```

Reply:
Could you explain the reasoning behind the suggestion? Grammatically it is fine to add a comma in there, and it emphasizes the reason more.

Reply:
I don't really want to have to do this all the time, but I would suggest throwing it up to a checker like https://quillbot.com/grammar-check. 

Specifically, you don't need a comma before the preposition here. https://prowritingaid.com/comma-after-prepositional-phrase

Reply:
Is it a mistake though? Comma usage can be very flexible. In this case, it is an infinitive phrase, which can be separated with commas depending on the meaning the author wants to emphasize. I know I am not a native English speaker, but let's not assume a mistake on my part if rules are ambiguous.

I want to keep the comment, as those lines look a bit out of place (since a lot of `SwitchPreference` are accessed in `onResume`) and the comment explains not what the code does, but why it has to be in the place it is.